JSONRPC/JSONRPC.cs

```diff
@@ -10,8 +10,8 @@ public class JSONRPC {
 	private string _endpoint;
 
 	// Basic authentication
-	private string _username;
-	private string _password;
+	private string _username = null;
+	private string _password = null;
```

Comment:
I do believe (by all means confirm) that strings in C# are initialized as `null`. I don't mind it if they're set here, but I do suggest you guys look into this.